src/github.com/getlantern/lantern-android/client/client.go

```diff
@@ -55,12 +53,14 @@ func NewClient(addr, appName string) *MobileClient {
 
 	hqfd := client.Configure(clientConfig.Client)
```

Comment:
Cool -- yeah I actually didn't realize go had that but it seems like it's mainly useful when you can't actually initialize something when it's declared -- somewhat akin to a static initializer block in java.